content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,411 @@
+
+## Install SPIRE
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SPIRE up and running:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml
+{{< /text >}}
+
+This will deploy SPIRE into your cluster, along with two additional components: the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) — used to share the SPIRE Agent's UNIX Domain Socket with the other
+pods throughout the node — and the [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar), a facilitator that performs automatic workload registration
+within Kubernetes. See [Install Istio](#install-istio) to configure Istio and integrate with the SPIFFE CSI Driver.
+
+### Option 2: Configure a custom SPIRE installation
+
+See the [SPIRE's Quick start for Kubernetes guide](https://spiffe.io/docs/latest/try/getting-started-k8s/)
+to get started deploying SPIRE into your Kubernetes environment. See [SPIRE CA Integration Prerequisites](#spire-ca-integration-prerequisites)
+for more information on configuring SPIRE to integrate with Istio deployments.
+
+#### SPIRE CA Integration Prerequisites
+
+To integrate your SPIRE deployment with Istio, configure SPIRE:
+
+1. Access the [SPIRE Agent reference](https://spiffe.io/docs/latest/deploying/spire_agent/#agent-configuration-file) and
+    configure the SPIRE Agent socket path to match the Envoy SDS defined socket path.
+
+    {{< text plain >}}
+    socket_path = "/run/secrets/workload-spiffe-uds/socket"
+    {{< /text >}}
+
+1. Share the SPIRE Agent socket with the pods within the node by deploying the
+    [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
+
+See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+
+{{< warning >}}
+Note that you must deploy SPIRE before installing Istio into your environment so that Istio can detect it as a CA.
+{{< /warning >}}
+
+## Install Istio
+
+1. [Download Istio release 1.14+](/docs/setup/getting-started/#download).
+
+1. After [deploying SPIRE](#install-spire) into your environment, and verifying that all deployments are in `Ready` state,
+    install Istio with custom patches for the Ingress-gateway as well as for istio-proxy.
+
+    {{< text bash >}}
+    $ istioctl install --skip-confirmation -f - <<EOF
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      namespace: istio-system
+    spec:
+      profile: default
+      meshConfig:
+        trustDomain: example.org
+      values:
+        global:
+        # This is used to customize the sidecar template
+        sidecarInjectorWebhook:
+          templates:
+            spire: |
+              spec:
+                containers:
+                - name: istio-proxy
+                  volumeMounts:
+                  - name: workload-socket
+                    mountPath: /run/secrets/workload-spiffe-uds
```

Comment:
@alexandrealvino Why it is /run/secrets/xxx instead of /var/run/secrets/xxx? Why remove the /var prefix? I know this /run/secrets/xxxx is aligned with spire-agent configmap configuration, but istio-agent only check /var/run/secrets/workload-spiffe-uds/socket as following shows, or I missed anything?
```go
    WorkloadIdentitySocketPath = "./var/run/secrets/workload-spiffe-uds/socket"
...
	socketExists, err := checkSocket(ctx, security.WorkloadIdentitySocketPath)
	if err != nil {
		return nil, fmt.Errorf("failed to check SDS socket: %v", err)
	}

	if socketExists {
		log.Info("SDS socket found. Istio SDS Server won't be started")
	} else {
		log.Info("SDS socket not found. Starting Istio SDS Server")
		err = a.initSdsServer()
		if err != nil {
			return nil, fmt.Errorf("failed to start SDS server: %v", err)
		}
	}
```

Reply:
`/var/run` is a symlink to `/run`

Reply:
ok, thanks for clarification.